static/js/containers/OrderSummaryPage.js

```diff
@@ -23,6 +23,9 @@ import type { RestState } from "../flow/restTypes"
 import type { CheckoutResponse } from "../flow/checkoutTypes"
 
 export const processCheckout = (result: CheckoutResponse) => {
+  if (!result) {
```

Comment:
I think this issue  #4823 had already beed fixed, or is this fixing something differnt?

Reply:
@annagav, You're right, These issues are connected (https://github.com/mitodl/micromasters/issues/4834 & https://github.com/mitodl/micromasters/issues/4823), The only difference is that the one you mentioned is related to the checkout API(Backend) and the one fixed in this PR is related to manipulation of the API Frontend response in the frontend.

We can keep this fix if we think the checkout API might give an error in the future, otherwise, we can remove it. I'm okay with both but I'll wait on you, either to remove it or keep it.